### DISCUSSION TITLE
MA archive doesn't show in HA Music Sources

### DISCUSSION THREAD

**Discussion by Kabraxist** [music-assistant/support#3030] [Q&A] [answered]
Hey there,
I was in the impression of I can access my Music Assistant archive directly from Home Assistant Music Sources interface.

Regarding Music Assistant, everything seems working normally. It's a Docker setup, I can use it to play music on my speakers, and HA Automations can control music with MA integration.

Did I got it wrong when I thought MA music will show up on my HA Music Sources?

**Comment by OzGav** [comment, marked as answer]
You have to select a MA player when you are looking at the HA media view

**Comment by Kabraxist** [comment]
Aaaah, I got it now!
I always thought that's a 'media player destination' and not a filter. Heh!

For other people confused like me:

![image](https://github.com/user-attachments/assets/600ad96b-8db5-431a-9f6e-2ad509d81fe8)

The dropdown here doesn't indicate a 'destination', but a device filter instead. You need to pick up one of your MA devices and your archive is right there.
